src/frontend/js/types/Joanie.ts

```diff
@@ -174,6 +174,8 @@ export enum OrderState {
   VALIDATED = 'validated',
 }
 
+export const activeOrderStates = [OrderState.PENDING, OrderState.VALIDATED, OrderState.SUBMITTED];
```

Comment:
By convention we are using uppercase and snake case to declare this kind of constant.

```
export const ACTIVE_ORDER_STATES = [OrderState.PENDING, OrderState.VALIDATED, OrderState.SUBMITTED];
```